packages/compass-settings/src/components/settings/gen-ai-settings.tsx

```diff
@@ -21,9 +21,12 @@ export const GenAISettings: React.FunctionComponent = () => {
       <SettingsList fields={['enableGenAIFeatures']} />
 
       {aiFeatureEnabled && (
-        <div className={atlasSettingsContainerStyles}>
-          <ConnectedAtlasLoginSettings></ConnectedAtlasLoginSettings>
-        </div>
+        <>
+          <div className={atlasSettingsContainerStyles}>
+            <ConnectedAtlasLoginSettings></ConnectedAtlasLoginSettings>
+          </div>
+          <SettingsList fields={['enableGenAISampleDocumentPassing']} />
```

Comment:
Going to update this setting to always show when the `enableGenAIFeatures` is enabled (even when it is not yet saved to true).

Reply:
I guess this is an outdated comment and we do it an a separate PR?

Reply:
Yup, we'll do this in https://jira.mongodb.org/browse/COMPASS-7800